sympy/matrices/tests/test_matrices.py

```diff
@@ -17,6 +17,9 @@
 from sympy.utilities.pytest import raises, XFAIL, slow, skip
 from sympy.solvers import solve
 from sympy.assumptions import Q
+import warnings
+from sympy.utilities.exceptions import SymPyDeprecationWarning
+warnings.filterwarnings("ignore", category=SymPyDeprecationWarning)
```

Comment:
You shouldn't need this line.  No test should call deprecated code.  If it does, there are still more places where the spelling needs to be changed.